### ISSUE TITLE
Bug: Borders in Firefox have "slices"

### ISSUE THREAD

**Issue by guastallaigor** [nostalgic-css/NES.css#273]
**Describe the bug**
refs #250 
This a different bug than it was before. It's hard to explain but in a different resolutions/components the borders has some slices on it.

**To Reproduce**
Open Firefox in the docs (`index.html`) or storybooks and check the borders of all the components.

**Expected behavior**
The borders are normal just like Chrome.

**Screenshots**
![image](https://user-images.githubusercontent.com/22016005/52384667-a03e1f00-2a65-11e9-9d62-e4bfeebe7194.png)

![image](https://user-images.githubusercontent.com/22016005/52384676-a46a3c80-2a65-11e9-8c3b-2d7c71ca5cbb.png)

![image](https://user-images.githubusercontent.com/22016005/52384684-ac29e100-2a65-11e9-9447-2f4b47c75574.png)

![image](https://user-images.githubusercontent.com/22016005/52384692-b0ee9500-2a65-11e9-84a6-4ab80b173140.png)

![image](https://user-images.githubusercontent.com/22016005/52384708-be0b8400-2a65-11e9-9e93-135fc5d2190e.png)

**Environment:**
* OS: Ubuntu 16.04
* Browser: Firefox Developer Edition 65.0b11 (64-bit) / Firefox Quantum 64.0 (64-bit) (test it on both)
* Build environment Node v10.15.0

**Suggestion(s) for fixing this issue**
It has something to do with `border-image`, but I'm not sure how to fix this yet.

**Additional context**
1. Happening only on Firefox;
2. The buttons that I put inside the docs in #272, they all have this bug. Also the "Copied!" Balloon has this bug as well.

**Comment by michaldaniel** [comment]
Seems like a result of property.

`border-image-repeat: space;`

The effect goes away on firefox with a change to.

`border-image-repeat: repeat;`

I'm not sure how helpful info that is. I came here a bit at random and started playing with inspector for a short while.

**Comment by BcRikko** [comment]
Thanks for your comment 👍 

I changed `border-image-repeat: space` to `repeat`.
Looks good 🎉 

But Chrome's display got weird. it's very difficult... 😱  

|#|chrome 72|firefox 65|
|---|---|---|
|`space`|![chrome-space](https://user-images.githubusercontent.com/5305599/52942330-26cff600-33ae-11e9-821e-51adb69e6cd4.png)|![ff-space](https://user-images.githubusercontent.com/5305599/52942334-2899b980-33ae-11e9-8837-a3713b1d7864.png)|
|`repeat`|![chrome-repeat](https://user-images.githubusercontent.com/5305599/52942351-30595e00-33ae-11e9-8c93-020209ad2ec0.png)|![ff-repeat](https://user-images.githubusercontent.com/5305599/52942357-33544e80-33ae-11e9-8e1b-aedb8e0e55e0.png)|

※ zoom: 110%

**Comment by BcRikko** [comment]
We might be able to resolve with using the following browser hacks. 🤔 
```css
/* only Firefox */
@supports (-moz-appearance: meterbar) {
  .nes-btn {
    border-image-repeat: repeat;
  }
}
```

**Comment by guastallaigor** [comment]
Closing this because it is fixed by #301.
